src/services/sports/index.jsx

```
@@ -0,0 +1,400 @@
+                      className={`px-2 ${bgColor} rounded-full text-white absolute top-0 left-1`}
+                    >
+                      Change Image
+                    </p>
+                    <img
+                      style={{ imageRendering: "-webkit-optimize-contrast" }}
+                      src={
+                        values.image instanceof File
+                          ? URL.createObjectURL(values?.image)
+                          : values.image
+                      }
+                      alt="Image"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)